data-processing/entities/abbreviations/types.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+
+from common.types import SerializableEntity
+
+
+@dataclass(frozen=True)
+class Abbreviation(SerializableEntity):
+    text: str
+    expansion : str
+    #the str_id tells us whether it is a full form or short form (f or s),
```

Comment:
Nit pick for comments style:
* one space between `#` and first character
* capitalize the first word
* end each one with a `.`